hardhat.config.ts

```diff
@@ -27,9 +27,39 @@ const {
   BSCSCAN_API_KEY,
   ARBISCAN_API_KEY,
   OPTIMISTIC_ETHERSCAN_API_KEY,
-  ZKSYNC_EXPLORER_API_KEY
+  ZKSYNC_EXPLORER_API_KEY,
+  VERSION,
+  ENV
 } = process.env;
 
+const version = VERSION ? VERSION.split('.')[0] : 'v1';
+
+const privateKey: { [key: string]: string } = {
+  v4Prod: PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT ?? '',
+  v4Test: PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT ?? '',
+  v2Prod: MAINNET_PRIVATE_KEY_FOR_CONTRACT_DEPLOYMENT ?? '',
+  v2Test: TESTNET_PRIVATE_KEY_FOR_CONTRACT_DEPLOYMENT ?? '',
+  v1ProdTestWallet: PRIVATE_KEY_FOR_V1_WALLET_CONTRACT_DEPLOYMENT ?? '',
```

Comment:
Do we use same private key for wallet and forwarder deployment in v1?

Because I have added something like this...
```
v1ProdTestWallet: PRIVATE_KEY_FOR_V1_WALLET_CONTRACT_DEPLOYMENT ?? '',
v1ProdTestForwarder: PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT_BACKUP ?? ''
```